src/common/task.h

```diff
@@ -37,7 +41,27 @@ class TaskExecutionSpec {
   /// Get the task spec size.
   ///
   /// @return The size of the immutable task spec.
-  int64_t SpecSize();
+  int64_t SpecSize() const;
+
+  /// Get the task's spillback count.
+  ///
+  /// @return The spillback count for this task.
+  int SpillbackCount() const;
+
```

Comment:
Can you document this method and also the above method, with a description of what the spillback count is?